lib/ClangImporter/ClangAdapter.cpp

```diff
@@ -712,7 +712,7 @@ OptionalTypeKind importer::getParamOptionality(version::Version swiftVersion,
     return OTK_None;
 
   // Check for the 'static' annotation on C arrays.
-  if (!swiftVersion.isVersion3())
+  if (swiftVersion != version::Version({3}))
```

Comment:
I would prefer to leave this unperturbed, which means back out this part of the change.  The idea is to just add the utility function isSwiftVersionLessThan for future code use and not perturb the existing code using isSwiftVersion3.